test/promql/tests/data_plane/gitops_sync_test.yaml

```diff
@@ -0,0 +1,665 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
+  # -----------------------------------------------------------------------------------
+  # The following section describes tests for the situation where no apps are Synced
+  # -----------------------------------------------------------------------------------
+  - interval: 1m
+    input_series:
+
+      - series: 'argocd_app_info{sync_status="Synced",name="gitopsdepl-001",namespace="gitops-service-argocd"}'
+        values: '_ _ _ _ _ _'
+      - series: 'argocd_app_info{sync_status="OutOfSync",name="gitopsdepl-002",namespace="gitops-service-argocd"}'
+        values: '1 1 1 1 1 1'
+      - series: 'argocd_app_info{sync_status="Unknown",name="gitopsdepl-003",namespace="gitops-service-argocd"}'
+        values: '1 1 1 1 1 1'
+
+    alert_rule_test:
+      - eval_time: 6m
```

Comment:
You have a look-back in the query of 5 minutes, and you're checking if it holds for 5 minutes. The eval time should be at least 10m.

Reply:
@yftacherzog , could you please elaborate on this?  The interval is 1m

Reply:
The query is looking at the last 5 minutes and the alert is triggering after 5 minutes of the query being met. 

So the query returns valid results only after 5 minutes and then you need that to hold for additional 5 minutes for the alert to kick in.

Reply:
Ok. I've widened the range, in the latest commit "Widen time series range". Resolving conversation.